tests/marqo_test.py

```diff
@@ -213,6 +216,58 @@ def setUpClass(cls) -> None:
+                    },
+                    {
+                        "indexName": cls.structured_index_name,
+                        "type": "structured",
+                        "allFields": [{"name": "text_field_1", "type": "text", "features": ["lexical_search", "filter"]},
+                                      {"name": "text_field_2", "type": "text", "features": ["lexical_search", "filter"]},
+                                      {"name": "text_field_3", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "int_field_1", "type": "int", "features": ["score_modifier"]},
+                                      {"name": "int_filter_field_1", "type": "int",
+                                       "features": ["filter", "score_modifier"]
+                                       }],
+                        "tensorFields": ["text_field_1", "text_field_2", "text_field_3"]
+                    },
+                    {
+                        "indexName": cls.unstructured_image_index_name,
+                        "type": "unstructured",
+                        "treatUrlsAndPointersAsImages": True,
+                        "model": "ViT-B/32",
+                    },
+                    {
+                        "indexName": cls.structured_image_index_name,
+                        "type": "structured",
+                        "allFields": [{"name": "text_field_1", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "text_field_2", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "text_field_3", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "image_field_1", "type": "image_pointer"},
+                                      ],
+                        "tensorFields": ["text_field_1", "text_field_2", "text_field_3", "image_field_1"],
+                        "model": "ViT-B/32",
+                    }
+                    # {
+                    #     "indexName": cls.structured_image_index_name_simple_preprocessing_method,
+                    #     "type": "structured",
+                    #     "allFields": [{"name": "text_field_1", "type": "text"},
+                    #                   {"name": "text_field_2", "type": "text"},
+                    #                   {"name": "text_field_3", "type": "text"}],
+                    #     "tensorFields": ["text_field_1", "text_field_2", "text_field_3"],
+                    #     "model": "ViT-B/16",
+                    #     "imagePreprocessingMethod": None,
+                    #     "treatUrlsAndPointersAsImages": True,
+                    # },
+                ])
+            except Exception as e:
+                print("Error creating indexes: ", e)
+
+
     @classmethod
     def tearDownClass(cls) -> None:
         """Delete commonly used test indexes after all tests are run
```

Comment:
If we use the same index settings for all the test classes, we don't have to create/delete indexes in setUpClass and tearDownClass even in open source.

Confirm with @pandu-k if this kind of setting is acceptable. On the one hand, we have fewer tests for index creation. On the other hand, we can leave that responsibility to API tests.

Reply:
@wanliAlex the open source tests should be more thorough in testing open source features and should be "purer" as they don't have to work around the constraints of setting up infrastructure in a remote cloud environment. 

The open source version of the tests should test marqo/py-marqo features more rigorously. the cloud versions are there to ensure the cloud is working as expected and that it isn't reducing any Marqo functionality